configs/mainnet.yaml

```diff
@@ -135,9 +135,12 @@ MESSAGE_DOMAIN_VALID_SNAPPY: 0x01000000
 SUBNETS_PER_NODE: 2
```

Comment:
Should `SUBNETS_PER_NODE` be removed while `SUBNETS_PER_SHARD` and `SHARDS_PER_NODE` be added instead?

Reply:
Have since removed the latter, but yes you are correct, thanks!